docs/configuration/index.md

```diff
@@ -650,25 +650,25 @@ When you specify an error response transform strategy other than `none`, Druid t
 ###### No error response transform strategy
 
 In this mode, Druid leaves error responses from underlying services unchanged and returns the unchanged errors to the API client.
-This is the default Druid error response mode. To explicitly enable this strategy, set `druid.server.http.errorResponseTransformStrategy.strategy` to "none".
+This is the default Druid error response mode. To explicitly enable this strategy, set `druid.server.http.errorResponseTransform.strategy` to "none".
 
 ###### Allowed regular expression error response transform strategy
 
-In this mode, Druid validates the error responses from underlying services against a list of regular expressions. Only error messages that match a configured regular expression are returned. To enable this strategy, set `druid.server.http.errorResponseTransformStrategy.strategy` to `allowedRegex`.
+In this mode, Druid validates the error responses from underlying services against a list of regular expressions. Only error messages that match a configured regular expression are returned. To enable this strategy, set `druid.server.http.errorResponseTransform.strategy` to `allowedRegex`.
 
 |Property|Description|Default|
 |--------|-----------|-------|
-|`druid.server.http.errorResponseTransformStrategy.allowedRegex`|The list of regular expressions Druid uses to validate error messages. If the error message matches any of the regular expressions, then Druid includes it in the response unchanged. If the error message does not match any of the regular expressions, Druid replaces the error message with null or with a default message depending on the type of underlying Exception. |`[]`|
+|`druid.server.http.errorResponseTransform.allowedRegex`|The list of regular expressions Druid uses to validate error messages. If the error message matches any of the regular expressions, then Druid includes it in the response unchanged. If the error message does not match any of the regular expressions, Druid replaces the error message with null or with a default message depending on the type of underlying Exception. |`[]`|
```

Comment:
I am a little confused. Based upon line 657, I thought it would be this:
`druid.server.http.errorResponseTransform.strategy=allowedRegex` . Do you need to change `strategy` to `allowedRegex` or would you append it in this notation.

Reply:
There are two different properties here, `druid.server.http.errorResponseTransform.strategy` and `druid.server.http.errorResponseTransform.allowedRegex`. The property `druid.server.http.errorResponseTransform.allowedRegex` will only matter if `druid.server.http.errorResponseTransform.strategy=allowedRegex`

Reply:
druid.server.http.errorResponseTransform.allowedRegex is the list of regular expression to match error message.
Example,
```
druid.server.http.errorResponseTransform.strategy=allowedRegex
druid.server.http.errorResponseTransform.allowedRegex=[".*CalciteException.*", ".*blahblahblah.*"]
```

Reply:
`druid.server.http.errorResponseTransform.strategy` is already described above (in line 641), hence it is not repeated here.